cdc/owner/changefeed.go

```diff
@@ -793,11 +793,12 @@ func (c *changefeed) releaseResources(ctx context.Context) {
 		c.cancel()
 	}
 	c.cancel = func() {}
+	c.wg.Wait()
 
 	if c.ddlPuller != nil {
 		c.ddlPuller.Close()
+		c.ddlPuller = nil
```

Comment:
Please add an unit test to ensure call `releaseResouces` double time will not panic or error anymore.